flocker/acceptance/scripts/reboot.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
Rename file, potentially to boot_id_service or somesuch.

I expected this to be a python file which rebooted the host it was run on, and was rather surprised that it merely reported boot_ids.